src/it/java/org/greencodeinitiative/creedengo/java/integration/tests/GCIRulesIT.java

```diff
@@ -548,4 +548,47 @@ void testGCI94() {
         checkIssuesForFile(filePath, ruleId, ruleMsg, startLines, endLines, SEVERITY, TYPE, EFFORT_1MIN);
     }
 
+    @Test
+    void testGCI96_1() {
```

Comment:
please update the test name with a more explcit name instead of "_1". this feedback is relevant to next test method names